PollyFill.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace GrayBMP;
+class PolyFill {
+   public void AddLine (int x0, int y0, int x1, int y1) {
+      var pt1 = (x0, y0); var pt2 = (x1, y1);
+      if (pt1 == pt2) return;
+      // Adjust segment ends along the sweep direction (at a lower y and then lower x).
+      if (Compare (pt1, pt2) > 0) (pt1, pt2) = (pt2, pt1);
+      Add (pt1); Add (pt2);
+
+      void Add ((int x, int y) pt) {

```

Comment:
The type `(int x, int y)` occurs several times in the code. It might be cleaner to just create a struct Point with this definition - performance won't change, but the code would be a bit easier to read. In general, Tuples are always a bit more difficult for humans to parse because the parentheses delimiting them could be easily confused with parentheses used for priority in expressions.

Reply:
Replaced the tuple with an integer base Point struct.